ci/build.yml

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+#
+# vim:set expandtab shiftwidth=2 tabstop=2:
+#
+# Variables:
+#   SpeechSdkVersion - Speech SDK version to use
+#   SpeechSdkNuGetSource - where to pick up the Speech SDK from ("nuget.org" or "VstsFeed")
```

Comment:
>or "VstsFeed" [](start = 77, length = 13)

is this public?

Reply:
ok - i see ;) just a url perhaps?

---
In reply to: [194070250](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/10#discussion_r194070250) [](ancestors = 194070250)

Reply:
for a VSTS feed, the associated build step actually wants a GUID. We may add additional optinos laters (e.g., pick up from build artifacts of another build).